core/src/test/java/io/questdb/griffin/AbstractO3Test.java

```diff
@@ -258,11 +278,9 @@ public int getO3ColumnMemorySize() {
                         return dataAppendPageSize > 0 ? dataAppendPageSize : super.getO3ColumnMemorySize();
                     }
                 };
-
-                TestUtils.execute(pool, runnable, configuration);
```

Comment:
Previously this branch was configuring O3 jobs on the provided worker pool (with the underlying `O3Utils#setupWorkerPool()` call). This doesn't happen anymore with this patch which seems to be an unintentional behavioral change in tests. We need to make sure to configure the pool properly here.

Reply:
It was intentional, I will put it in there now to show it is the same behaviour.